### DISCUSSION TITLE
How to migrate from Cobalt CCK ?

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [JoomCoder-com/JoomCCK#117] [Q&A] [answered]
How to migrate from Cobalt 8?
Ist it possible to test before we buy?

**Comment by andiamine** [comment]
I will write an article about it this week.
JoomCCK is free you can download it and test it.

**Comment by LordHelmchen666** [comment]
Just to be clear. I am willing to buy a subscription, but before i will be sure i can import from Cobalt and then use JoomCCK as i need it. ;-)
For free version i have to download source from git and then build it myself?

**Reply by andiamine** [reply]
You don't have to build it, you can download last update from here:
https://github.com/JoomCoder-com/JoomCCK/releases

**Reply by LordHelmchen666** [reply]
I see only Sourcecode packages. No Joomla install packages.

**Reply by andiamine** [reply]
yes you click on it to download it and install it as any joomla extension.

**Reply by LordHelmchen666** [reply]
No i can not, because this packages are not installable as any Joomla extension. Try it and you get a message "no installfile found". ;)

**Reply by andiamine** [reply]
I do apologize i did forgot to attach file.

**Comment by Giorgi625** [comment]
Agree, only beta versions have installable packages, stable version has only source files.

**Reply by andiamine** [reply]
Now is available sorry.

**Reply by LordHelmchen666** [reply]
No problem. We all just humans. ;)

**Comment by LordHelmchen666** [comment]
Any chance to get an "Migration how to"?
I can't wait to start the migration. And Joomla 3 has reached EOL. :(

**Reply by andiamine** [reply]
we will write a small tutorial of steps to follow in the beginning of the new year. 

thanks for your patience.

**Comment by Kraftion** [comment]
Has anyone managed to transfer the site from Cobalt to JoomCCK?

**Reply by andiamine** [reply]
we will write a small tutorial of steps to follow in the beginning of the new year. 

thanks for your patience.

**Reply by Kraftion** [reply]
Hello
Any news about migration process?

**Comment by Giorgi625** [comment]
No, Andi has not still provided tutorial. I just started recreating from scratch my old projects on J5 and JoomCCK, but even though There are problems with Map field and Records module so I am stuck in the middle :(

**Reply by andiamine** [reply]
Hi my friend, i do apologize to let you wait, team will start working on JoomCCK and JoomSubscription next week. please hold on.

regards,
ANDI

**Comment by LordHelmchen666** [comment]
If there won't be a migration path soon i will drop all my Cobalt sites and don't use JoomCCK. Because PHP and Joomla i need to run Cobalt is outdated.

**Reply by LordHelmchen666** [reply]
I tried to update from J3 to J5 and it was succesfull.

1. Make an SQL dump from Cobalt 8 install.  (all tables with js_res_***)
2. Uninstall Cobalt 8
3. Upgrade J3 to J4 or J5.
4. Install JoomCCK in J4 or J5.
5. Import the SQL dump in J4/J5 install.

Then you need to update your menu of course. Also need to resave most settings of templates, fields, etc. if something not working.

My J3 site https://helmut-mueller-online.de/hoerbuchtipps.html
J5 site https://joomla.müllerhelmut.de/index.php?option=com_joomcck&view=records&section_id=6:hoerbuchdatenbank&Itemid=742 (not everything is complete)

**Comment by Giorgi625** [comment]
I noticed that in Full article template JoomCCK uses new code for Record title and icons with settings dropdown menu, so you will need to edit it if you use template from Cobalt CCK, otherwise title will show up but dropdown menu will not work.

**Comment by andiamine** [comment, marked as answer]
Hi everyone, please refer to this article about migration
https://github.com/JoomCoder-com/JoomCCK/wiki/Migration#migration-from-cobalt-8-by-mintjoomla

**Reply by Anri0303** [reply]
Good afternoon, could you give instructions on how to work with the database, how to select and insert these tables. I haven't had any such work experience, and attempts don't succeed. website poiskvodoema.com I want to migrate to joomla 4 
 recover. Help please.

**Reply by Anri0303** [reply]
How to export all Cobalt v8+ tables as an SQL file using phpMyAdmin (all tables start with #_js*)?

**Reply by andiamine** [reply]
use PHPMyAdmin has all features to export and to rename tables.
